test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,53 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+
+const urlBase = 'https://api.github.com/user';
```

Comment:
we can use better names to make the code easy to read, in this case, this can be renamed as `userUrl` or `userApiUrl`